### PR TITLE
feat: add tokenizer for number

### LINKED ISSUE #2: feat: add number tokenizer

### PR DESCRIPTION
**Pull request by khang00**
Purpose:
 - Resolves #2 

Changes:
 - add tokenizer for number
 - add basic token structure

### CODE REVIEW

File: src/ast/tokenizer.rs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        } else {
+            return Err(TokenizerError {
+                error_type: ErrorType::NumberError,
+            });
+        }
+    }
+
+    Ok(Number(number))
+}
+
```
Comment (SuperBo) [inline src/ast/tokenizer.rs]: please add  #[cfg(test)]

File: src/ast/token.rs [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    /// Less Than Or Equals operator `<=`
+    LtEq,
+    /// Greater Than Or Equals operator `>=`
+    GtEq,
+    /// Period (used for compound identifiers or projections into nested types)
+    Period,
+    /// Comma `,`
+    Comma,
+    /// SemiColon `;` used as separator for COPY and payload
+    SemiColon,
```
Comment (SuperBo) [inline src/ast/token.rs]: Should we add quote character "`". This is used in Bigquery.

File: src/ast/tokenizer.rs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+use nom::character::is_digit;
+use crate::ast::token::{ErrorType, Token, TokenizerError};
+use crate::ast::token::Token::Number;
+
+pub fn tokenize_number(text: &String) -> Result<Token, TokenizerError> {
+    let mut number = String::from("");
+    for &item in text.as_bytes().iter() {
+        if is_digit(item) {
```
Comment (SuperBo) [inline src/ast/tokenizer.rs]: Are you considering floating number yet?

**Review by SuperBo — COMMENTED** [review]
Please see my comments

File: src/ast/token.rs (hunk 2 above)
Comment (khang00) [reply src/ast/token.rs]: leave it for later

**Review by SuperBo — APPROVED** [review]

**Pull request merged**